examples/uptime_sla/test/UptimeSLA_test.js

```diff
@@ -7,33 +7,34 @@ import {
 } from './support/helpers'
 import {
   assertActionThrows,
+  bigNum,
   eth,
+  deploy,
   functionSelector,
-  getEvents,
   getLatestEvent,
   oracleNode,
   requestDataBytes,
-  requestDataFrom
+  requestDataFrom,
+  toWei,
 } from '../../../solidity/test/support/helpers'
-import { assertBigNum } from '../../../solidity/test/support/matchers'
```

Comment:
What's the motivation behind replacing the matcher with many invocations of `bigNum(...)`? Seems like DRYing it up in a central location is a more manageable solution. If we ever need to make a change, we don't need to tweak the N number of `bigNum` calls. Same goes for the `.toString()` invocations.

Reply:
> DRYing it up in a central location is a more manageable solution

Yeah, I'll do that.

> Same goes for the .toString() invocations.

The issue there is that contracts don't accept `BN`s for `uint256` arguments, etc. We are going to face some difficulty like this somewhere at the moment until truffle contracts catch up, I think. (I'm sure they'd take a patch.) Before I had `transferLINK` and `transferLINKAndCall` functions in `helpers.js`, but @se3000 asked me to change them back to flat `link.transfer*` calls with `.toString` conversions.

Reply:
Ah, interesting. Well, if the intent was to coerce the calls to `string` or `uint256`, let's make that clear.

Perhaps we can make a `link` wrapper that looks identical to the existing `link` (same name, different import) with the sole exception that it coerces the return value to the correct type.